File: projeto incorreto/prontuarioEletronico-main/java/src/Paciente.java
public class Paciente {
    private String nome;
    private String cpf;
    private String telefone;
    private String endereco;
    private String nascimento;

    public Paciente(String nome, String cpf, String telefone, String endereco, String nascimento) {
        this.nome = nome;
        this.cpf = cpf;
        this.telefone = telefone;
        this.endereco = endereco;
        this.nascimento = nascimento;
    }

    public String getNome() { return nome; }
    public String getCpf() { return cpf; }
    public String getTelefone() { return telefone; }
    public String getEndereco() { return endereco; }
    public String getNascimento() { return nascimento; }

    public void setNome(String nome) { this.nome = nome; }
    public void setTelefone(String telefone) { this.telefone = telefone; }
    public void setEndereco(String endereco) { this.endereco = endereco; }
    public void setNascimento(String nascimento) { this.nascimento = nascimento; }

    @Override
    public String toString() {
        return "Nome: " + nome + " | CPF: " + cpf + " | Telefone: " + telefone +
               " | Endereço: " + endereco + " | Nascimento: " + nascimento;
    }
}

File: projeto incorreto/prontuarioEletronico-main/php/models/Paciente.php

<?php
class Paciente {
    private $nome;
    private $nascimento;
    private $cpf;
    private $sexo;
    private $telefone;
    private $email;
    private $endereco;
    private $convenio;
    private $observacoes;

    public function __construct($nome, $nascimento, $cpf, $sexo, $telefone, $email, $endereco, $convenio, $observacoes) {
        $this->nome = $nome;
        $this->nascimento = $nascimento;
        $this->cpf = $cpf;
        $this->sexo = $sexo;
        $this->telefone = $telefone;
        $this->email = $email;
        $this->endereco = $endereco;
        $this->convenio = $convenio;
        $this->observacoes = $observacoes;
    }

    public function getNome() {
        return $this->nome;
    }

    public function getCpf() {
        return $this->cpf;
    }

    public function toArray() {
        return [
            'nome' => $this->nome,
            'nascimento' => $this->nascimento,
            'cpf' => $this->cpf,
            'sexo' => $this->sexo,
            'telefone' => $this->telefone,
            'email' => $this->email,
            'endereco' => $this->endereco,
            'convenio' => $this->convenio,
            'observacoes' => $this->observacoes
        ];
    }
}
?>